packages/dd-trace/src/profiling/config.js

```diff
@@ -191,6 +193,44 @@ class Config {
     logExperimentalVarDeprecation('CPU_ENABLED')
     checkOptionWithSamplingContextAllowed(this.cpuProfilingEnabled, 'CPU profiling')
 
+    const uploadCompression0 = coalesce(options.uploadCompression, DD_PROFILING_DEBUG_UPLOAD_COMPRESSION, 'on')
+    let [uploadCompression, level0] = uploadCompression0.split('-')
+    if (!['on', 'off', 'gzip', 'zstd'].includes(uploadCompression)) {
```

Comment:
What would you think of using 'default' instead of 'on' ?

Reply:
There's a notebook attached to the Jira issue that specified the recognized values as these four.

Reply:
Ok, I missed that for once we tried to standardize across profilers